source/-html/SinglePageVisitor.cpp

```diff
@@ -0,0 +1,101 @@
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Copyright (c) 2023 Vinnie Falco (vinnie.falco@gmail.com)
+//
+// Official repository: https://github.com/cppalliance/mrdox
+//
+
+#include "SinglePageVisitor.hpp"
+#include <mrdox/Support/unlock_guard.hpp>
+
+namespace clang {
+namespace mrdox {
+namespace html {
+
+// Launch a task to render the page
+// pageNumber is zero-based
+//
+void
+SinglePageVisitor::
```

Comment:
You might have a look at the adoc generator again, I have made some nice improvements to this code.